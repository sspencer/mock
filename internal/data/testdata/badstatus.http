### Bad status
# @status 999
DELETE /bread/1
