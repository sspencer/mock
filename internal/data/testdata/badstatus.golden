invalid status, line 2: # @status 999